http/server.ts

```diff
@@ -773,8 +773,10 @@ export async function serveTls(
     once: true,
   });
 
-  const key = options.key || await Deno.readTextFile(options.keyFile!);
-  const cert = options.cert || await Deno.readTextFile(options.certFile!);
+  // deno-lint-ignore no-sync-fn-in-async-fn
```

Comment:
I think this is ok since the `serveTls` function is deprecated and will be removed soon

Reply:
Usually, we shouldn't bother. But this is a small/simple change. Can you please use `await Deno.readTextFile()` here?

Reply:
I tried but it seems to break the test https://github.com/denoland/deno_std/actions/runs/8232529366/job/22510162053#step:4:5048

Reply:
Oh, that's interesting. @bartlomieju, do you have any idea why this might be? Is this expected behaviour? This specific instance doesn't matter much; rather, the behaviour surprises me.

Either way, @gabelluardo, let's ignore 👍🏾